app/models/cdo/edit/exemption-details.js

```diff
@@ -9,6 +9,25 @@ function ViewModel (exemption, courts, policeForces, companies, backNav, errors)
     srcHashParam: backNav.srcHashParam,
     exemption: {
       indexNumber: exemption.indexNumber,
+      exemptionOrder: {
+        label: {
+          text: 'Order',
+          classes: 'govuk-!-font-weight-bold govuk-!-font-size-16'
+        },
+        name: 'exemptionOrder',
+        id: 'exemptionOrder',
+        value: exemption.exemptionOrder,
+        items: [
+          {
+            value: '2015',
```

Comment:
These values should ideally be coming from the DB since we now have a DB table - but as only 2 values I suppose it's fine